cmd/controller/main.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/aws/karpenter/pkg/cloudprovider/registry"
 	"github.com/aws/karpenter/pkg/controllers"
 	"github.com/aws/karpenter/pkg/controllers/counter"
-	"github.com/aws/karpenter/pkg/controllers/metrics"
+	"github.com/aws/karpenter/pkg/controllers/metrics/podmetrics"
```

Comment:
Don't let the name collision force your package naming elsewhere. Consider something like 
```suggestion
	metricspod "github.com/aws/karpenter/pkg/controllers/metrics/pod"
```

Reply:
done